src/Pages/Main/Payment/MethodList.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+
+class MethodList extends Component {
+  state = {
+    active: true,
+  };
+
+  getIndex = () => {
+    const { index } = this.props;
+    console.log(this.props.index);
+    if (
+      index === 0 ||
+      index === 1 ||
+      index === 2 ||
+      index === 3 ||
+      index === 4
+    ) {
+      this.setState({
+        active: !this.state.active,
+      });
+    }
+  };
+
+  render() {
+    const { active } = this.state;
+    const { img, text, handlechangeActive } = this.props;
+    const { getIndex } = this;
+    return (
+      <div className="MethodList">
+        <div
+          className="list-light-index"
+          // onClick={handlechangeActive(this.props.index)}
+          onClick={getIndex}
+        >
+          <div className={active ? "option-list" : "option-list-active"}>
```

Comment:
```suggestion
          <div className={`option-list ${!active ? "active" : ""}`}>
```
원래 코드와 지금 코드간에 어떤 차이가 있을까요?

Reply:
없습니다 ... 
버튼 on /off 기능 추후 정리해서 다시 푸쉬 하겠습니다 ㅠ

Reply:
더 간결해진것(?)
맞나요 😳

Reply:
아니면 state값을 변경해줘서 거기에 따른 클래스명 변경!???🙄

Reply:
@Bhoon-coding 기존에는 option-list, option-list-active 통째로 별도로 스타일링 했죠. 중복되는 스타일이 있더라도. 반면 후자의 경우는 option-list라는 공통속성을 쓰고 변화하는 active 속성만 변경해주는 방식입니다.

Reply:
갓택... 😨